### PR TITLE
Rework Phase 3 Curriculum pages

### PR DESCRIPTION
**Pull request by mikecargal**
Aligning Phase 3 curriculum with goal of working through The Odin Project Full Stack JavaScript path.

Adding template repositories and project plans to assist tracking.

### CODE REVIEW

**Review by ashley — CHANGES_REQUESTED** [review]
Some minor requests, but overall, the workflow and project setup lgtm.

File: docs/Phase-3/phase-3.md
[the remark this answers is not in the document]
Comment (mikecargal) [reply docs/Phase-3/phase-3.md:1]: I need to remove mention of modules. I thought I had gotten them, but that’s probably why I needed to step back and look at it with fresh eyes after a couple of days.

File: docs/Phase-3/project-workflow.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Curriculum Project Workflow
+nav_order: 2
+has_children: false
+parent: Phase 3 - Intro to Programming
+permalink: /phase3/projectworkflow
+---
+
+# Curriculum Project Workflow
+
+This project workflow is designed to assist in tracking progress through The Odin Project's JavaScrip course,
```
[the remark this answers is not in the document]
Comment (mikecargal) [reply docs/Phase-3/project-workflow.md:12]: Details, details…

File: docs/Phase-3/repository-and-project-setup.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    <br/><img srcset="/curriculum/assets/images/UseTemplate2.png 2x">
+  - Click the "Create repository from template" button
+
+## Set up your project
+
+- Go to the matching project.
+  - For course 1 (Intermediate HTML and CSS) use [EmergentWorks Phase 3 Course 1 Tracking](https://github.com/orgs/emergentworks/projects/4)
+  - For course 2 (JavaScript) use [EmergentWorks Phase 3 Course 2 Tracking](https://github.com/orgs/emergentworks/projects/2)
+- Click on the ellipses on the right, and click "Make a copy"
+  <br/><img srcset="/curriculum/assets/images/CopyProject1.png 2x">
```
[the remark this answers is not in the document]
Comment (mikecargal) [reply docs/Phase-3/repository-and-project-setup.md:37]: Ah, I always do it from the projects list, but this is probably less error prone.  TBH, I don’t think I had even noticed the option on that menu 🙄

**Comment by mikecargal** [comment]
I just noticed that I need to rework the “Learning Topics” on the Summary page for Phase 3.  (It just has the JavaScript course).  I’ll rework that before as well

**Comment by mikecargal** [comment]
OK, I've taken care of your comments (thanks!), and I've fleshed out the summary page to reflect the full Full Stack JavaScript path in TOP.  I think it's ready to merge.

(I've only fleshed out repo templates and projects for the first two courses.  I'll just need to stay a step ahead of mentees, but I REALLY don't see any making it through the first two course in 12 weeks.... If I'm wrong, I'll gladly scramble to flesh out those items)

**Review by ashley — APPROVED** [review]

**Pull request merged**